### PR TITLE
Make Polish translation more consistent

### PR DESCRIPTION
**Pull request by mkljczk**
Signed-off-by: Marcin Mikołajczak <me@m4sk.in>

### CODE REVIEW

File: lxqt-config/lxqt-config-appearance/lxqt-config-appearance_pl_PL.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="pl_PL">
```
Comment (palinek) [inline lxqt-config/lxqt-config-appearance/lxqt-config-appearance_pl_PL.ts]: why did you decrease the version number?

File: lxqt-globalkeys/lxqt-config-globalkeyshortcuts/lxqt-config-globalkeyshortcuts_pl_PL.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@
     </message>
     <message>
         <source>Find a command</source>
-        <translation type="vanished">Znajdź polecenie</translation>
```
Comment (palinek) [inline lxqt-globalkeys/lxqt-config-globalkeyshortcuts/lxqt-config-globalkeyshortcuts_pl_PL.ts]: "vanished" means, that this text has gone and is not used any more

File: obconf-qt/obconf-qt_pl.ts [hunk 3]
```diff
@@ -771,7 +771,9 @@ Opcje:
         <source>Error while parsing the Openbox configuration file.  Your configuration file is not valid XML.
 
 Message: %1</source>
-        <translation>Błąd wczytania pliku rc.xml. Prawdopodobnie Openbox nie został poprawnie zainstalowany.</translation>
+        <translation>Błąd wczytania pliku rc.xml. Prawdopodobnie Openbox nie został poprawnie zainstalowany.
```
Comment (palinek) [inline obconf-qt/obconf-qt_pl.ts]: IMO "Your configuration file is not valid XML." definitely isn't "Openbox nie został poprawnie zainstalowany."

File: lxqt-config/lxqt-config-appearance/lxqt-config-appearance_pl_PL.ts (hunk 1 above)
Comment (mkljczk) [reply lxqt-config/lxqt-config-appearance/lxqt-config-appearance_pl_PL.ts]: Linguist did it for me…

**Comment by mkljczk** [comment]
now should be ok

**Comment by palinek** [comment]
merged to master

**Pull request closed without merging**